### ISSUE TITLE
static latest version address

### ISSUE THREAD

**Issue by MHA15** [greggman/twgl.js#64]
hi
I think that you are separating dist versions in N.x directories for backward compatibility but please add another directory for latest version which the latest version being there always.
thanks

**Comment by greggman** [comment]
I don't think it would be good to do that. The whole point of major version numbers is they break something. So anyone using a "latest" link is going to break when a major version is pushed. 

Basically you need opt into using a major version change. If you're using the it through npm or browserify (not even sure that's still a thing), then it doesn't matter. If you opt into a new major version update by updating your dependencies you'll get the latest as it's referenced in `package.json`.  If you're linking some other way then you'd just be asking to get randomly broken.

The numbering scheme is [semver](https://semver.org)

> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.
